plutus-core/plutus-core/src/PlutusCore/Builtin/KnownType.hs

```diff
@@ -373,24 +376,9 @@ instance HasConstantIn uni val => ReadKnownIn uni val (SomeConstant uni rep) whe
     {-# INLINE readKnown #-}
 
 instance uni ~ UniOf val => MakeKnownIn uni val (Opaque val rep) where
-    makeKnown = coerceArg pure  -- A faster @pure . Opaque@.
+    makeKnown = coerceArg pure
```

Comment:
We now have lots of these `coerceArg` calls, so no point explaining each of them, given that the name is pretty suggestive and the user can jump-to-definition, if they want more info.

Wasn't correct anyway, it was a faster `pure . unOpaque`, not `pure . Opaque`.